core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -96,6 +96,8 @@ object Controller {
     }
 
     def main(args: Array[String]): Unit = {
+        implicit val system = ActorSystem("controller-actor-system")
```

Comment:
> I believe this changes the behavior if the config is not valid? (hangs because no one terminates the ActorSystem)
> 
I concur. It should get terminated. The controller will not be healthy and won't respond to ping though.